qase-javascript-commons/src/options/options-type.ts

```diff
@@ -23,19 +23,21 @@ export type AdditionalReportOptionsType = {
 };
 
 export type OptionsType = {
-  frameworkPackage: string;
-  frameworkName: string;
-  reporterName: string;
-  mode?: `${ModeEnum}` | undefined;
-  fallback?: `${ModeEnum}` | undefined;
+  captureLogs?: boolean | undefined;
   debug?: boolean | undefined;
   environment?: string | number | undefined;
+  fallback?: `${ModeEnum}` | undefined;
+  frameworkName: string;
+  frameworkPackage: string;
+  mode?: `${ModeEnum}` | undefined;
+  report?: RecursivePartial<AdditionalReportOptionsType> | undefined;
+  reporterName: string;
```

Comment:
I see many changed lines here, but most of them were just reordered. What is the meaning of this change?

Reply:
I've removed the ordering changes